@shared/api/types.ts

```diff
@@ -45,12 +45,12 @@ export interface Response {
   recentAddresses: Array<string>;
   hardwareWalletType: WalletType;
   bipPath: string;
-  blockedDomains: Array<BlockedDomain>;
+  blockedDomains: BlockedDomains;
+  assetDomain: string;
 }
 
-export interface BlockedDomain {
-  domain: string;
-  paging_token: string;
+export interface BlockedDomains {
```

Comment:
I changed BlockedDomains to be a map w/ the domain as the key. Makes it much easier to use than an array of objects